src/wallet/rpcdump.cpp

```diff
@@ -1137,12 +1137,12 @@ UniValue importmulti(const JSONRPCRequest& mainRequest)
                 // range, or if the import result already has an error set, let
                 // the result stand unmodified. Otherwise replace the result
                 // with an error message.
-                if (GetImportTimestamp(request, now) - TIMESTAMP_WINDOW >= scannedRange->GetBlockTimeMax() || results.at(i).exists("error")) {
+                if (GetImportTimestamp(request, now) - TIMESTAMP_WINDOW > scanFailed->GetBlockTimeMax() || results.at(i).exists("error")) {
```

Comment:
> because all the blocks that were bad or missing are older than the key, and therefore not relevant.

That's the part I'm not sure about: aren't the blocks *at and after* that point missing, instead of before that point?